src/components/logout/Logout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import vector from '../../shared/images/vector.svg';
+
+function Logout() {
+  const navigate = useNavigate();
+
+  const callback = () => {
+    navigate('/sign_in');
```

Comment:
Can you use const Paths.sign_in that is defined in 'path' folder (like in Routing)?

Reply:
Of course!